apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -267,15 +259,14 @@ const aichatSlice = createSlice({
   initialState,
   reducers: {
     addChatMessage: (state, action: PayloadAction<ChatCompletionMessage>) => {
-      const newMessageId =
-        state.chatMessages[state.chatMessages.length - 1].id + 1;
+      const newMessageId = state.chatMessages.length + 1;
```

Comment:
>Is this change needed for this PR?

Since `initialChatMessage` is now empty, I had to update this line so that we didn't get an error the first time this is assigned.
I also realized that `id` was assigned to a user message in both `submtChatContents` and then reassigned here so made `id` in `ChatCompletionMessage` optional so can be assigned only once.

>I think this might cause issues when you delete a single status message from the chat window?

This doesn't seem to be causing an issue when I delete update status messages.

Reply:
Actually, I see it can. Thanks! will rethink.

Reply:
Created a helper function to get `lastMessageId` and assign `id` to `ChatCompletionMessage` before dispatching `addChatMessage`. This may end up being moot if we assign `id` after getting response from backend but works for now. Thanks!